core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -147,4 +142,51 @@ object JsonEditor {
 
     doUpdateWorkflow(workflowJson = json)
   }
+
+  /**
+    * Update workflow json, replacing subWorkflowMetadata elements of root workflow's "calls" object by subworkflowId
+    *
+    * @param workflowJson json blob
+    * @return updated json
+    */
+  def replaceSubworkflowMetadataWithId(workflowJson: Json): Json = {
+    def replaceSubworkflowMetadataObjectWithSubworkflowIdInCalls(callObject: JsonObject, subworkflowJson: Json): Json = {
+      val callObjectWithRemovedSubworkflow = callObject.remove(subWorkflowMetadataKey)
+      val resultingCallObject = subworkflowJson.rootWorkflowId match {
```

Comment:
The `rootWorkflowId` method needs to be renamed to something like `id` (yes I'm the person who originally named it 🙂). It actually picks out the value for the `id` key which is probably surprising when it is applied to a subworkflow JSON rather than the root workflow JSON.

Reply:
I think `workflowId` would be more suitable name than just `id`, since it returns value of `WorkflowId` type.

Reply:
sounds good